SU-SWS/sdss_profile
# SDSS-167: fixing basic page teaser. Removed Italics from SDSS-151

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- SDSS-167: made link dark green and weight 400
- SDSS-151: made splash regular from italic font

# Review By (Date)
- soon

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to http://sdss.loc/qa-list-and-teaser-display?check_logged_in=1
4. Verify the QA Notes link is weight 400 instead of bold
5. Navigate to http://sdss.loc/qa-typography-styles
6. Verify the Splash text is regular font and not italics

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [SDSS-167](https://stanfordits.atlassian.net/browse/SDSS-167)
- [SDSS-151](https://stanfordits.atlassian.net/browse/SDSS-151)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
